pkg/crypto.go

```diff
@@ -145,6 +146,49 @@ func (k opaquePrivateKey) Sign(rand io.Reader, digest []byte, opts crypto.Signer
 	return k.signFn(rand, digest, opts)
 }
 
+// Start the certificate monitors
+func (client *Crypto) Start() error {
+	periods := []struct {
+		period time.Duration
+		label  string
+	}{
+		{
+			time.Hour * 24,
+			"day",
+		},
+		{
+			time.Hour * 24 * 7,
+			"week",
+		},
+		{
+			time.Hour * 24 * 7 * 4,
+			"4_weeks",
+		},
+	}
+
+	for _, p := range periods {
+		m := storage.CertificateMonitor{
+			Period:      p.period,
+			PeriodLabel: p.label,
+			Storage:     client.Storage,
+		}
```

Comment:
isn't this something to move to the `storage` package as `DefaultCertificateMonitors` or something?

Or at least an initialization function in `storage` (`NewCertificateMonitor(...)`)

Reply:
moved it to DefaultCertificateMonitors and made attrs private